lib/terraforming/resource/iam_group_membership.rb

```diff
@@ -44,7 +44,8 @@ def tfstate
       private
 
       def group_members_of(group)
-        @client.get_group(group_name: group.group_name).users.map { |user| user.user_name }
+        user_query = @client.get_group(group_name: group.group_name)
+        user_query.map { |r| r.users }.flatten.map { |user| user.user_name }
```

Comment:
Sorry, maximum limitation of lines is misconfigured... 🙇 
Please write this in one line.

```ruby
@client.get_group(group_name: group.group_name).map(&:users).flatten.map(&:user_name)
```